tests/playwright/src/model/bootc-page.ts

```diff
@@ -0,0 +1,158 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+import { waitUntil, waitWhile } from '@podman-desktop/tests-playwright';
+import { ArchitectureType } from '@podman-desktop/tests-playwright';
+
+export class BootcPage {
+  readonly page: Page;
+  readonly webview: Page;
+  readonly heading: Locator;
+  readonly outputFolderPath: Locator;
+  readonly rawCheckbox: Locator;
+  readonly qcow2Checkbox: Locator;
+  readonly isoCheckbox: Locator;
+  readonly vmdkCheckbox: Locator;
+  readonly amiCheckbox: Locator;
+  readonly amd64Button: Locator;
+  readonly arm64Button: Locator;
+  readonly buildButton: Locator;
+  readonly imageSelect: Locator;
+  readonly goBackButton: Locator;
+  readonly rowGroup: Locator;
+  readonly latestBuiltImage: Locator;
+  readonly getCurrentStatusOfLatestBuildImage: Locator;
+
+  constructor(page: Page, webview: Page) {
+    this.page = page;
+    this.webview = webview;
+    this.heading = webview.getByLabel('Build Disk Image');
+    this.outputFolderPath = webview.getByLabel('folder-select');
+    this.imageSelect = webview.getByLabel('image-select');
+    this.rawCheckbox = webview.locator('label[for="raw"]');
```

Comment:
looks like we could tackle ARIA functionality on the extension side? Please create an issue. Then we could use getByRole instead of locator.